pkg/util/ecs/common.go

```diff
@@ -110,18 +117,33 @@ func convertMetaV2Container(c v2.Container) *containers.Container {
 		container.StartedAt = startedAt.Unix()
 	}
 
-	if l, found := c.Limits["cpu"]; found && l > 0 {
+	if l, found := c.Limits[cpuKey]; found && l > 0 {
 		container.Limits.CPULimit = float64(l)
+	} else if l, found := taskLimits[cpuKey]; found && l > 0 {
+		container.Limits.CPULimit = formatTaskCPULimit(l)
 	} else {
 		container.Limits.CPULimit = 100
 	}
-	if l, found := c.Limits["memory"]; found && l > 0 {
-		container.Limits.MemLimit = l
+
+	if l, found := c.Limits[memoryKey]; found && l > 0 {
+		container.Limits.MemLimit = formatMemoryLimit(l)
+	} else if l, found := taskLimits[memoryKey]; found && l > 0 {
+		container.Limits.MemLimit = formatMemoryLimit(uint64(l))
 	}
 
 	return container
 }
 
+func formatTaskCPULimit(val float64) float64 {
+	// The ECS API exposes the task CPU limit with the format: 0.25, 0.5, 1, 2, 4
+	return val * 100
```

Comment:
I think container limits are expressed in shares, which is 1 CPU = 1024 Shares, it'd expect the task CPU Limit to be in # of CPUs.

Reply:
for the process agent 1 vCPU == 100

Reply:
PR update to convert the task limit to CPU units which corresponds to what customers configure in their tasks and see on the ECS UI, curious to know what @DataDog/processes think about this.

Reply:
We do have the notion of `1 vCPU == 100` so it makes sense. Thanks for adding the comment.